applications/GeoMechanicsApplication/python_scripts/geomechanics_U_Pw_solver.py

```diff
@@ -119,6 +119,7 @@ def AddDofs(self):
 
         ## Fluid dofs
         KratosMultiphysics.VariableUtils().AddDof(KratosMultiphysics.WATER_PRESSURE, KratosMultiphysics.REACTION_WATER_PRESSURE,self.main_model_part)
+        KratosMultiphysics.VariableUtils().AddDof(KratosGeo.DT_WATER_PRESSURE, self.main_model_part)
```

Comment:
I added this DoF here, since for some reason, after adding this test, I got the following error for the prescribed derivative: 
![image](https://github.com/KratosMultiphysics/Kratos/assets/56549273/0bbf41a9-498b-463e-87ab-d0d267e3155a)

Turns out we don't add the dof here, but still fix it in the node. How this has ever worked, I do not know.

Reply:
See my previous comment.
The code of the solvers largely is copied. In future we may consider removing the duplications.